src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let summary = 0;
+
+  return function makeAdder(number) {
+    if (number === undefined) {
+      const currentSummary = summary;
```

Comment:
```suggestion
      const currentSum = sum;
```

i thought the word summary is used to describe something different, not a sum of two numbers